sound/soc/intel/common/soc-acpi-intel-mtl-match.c

```diff
@@ -715,6 +756,31 @@ static const struct snd_soc_acpi_link_adr cs42l43_link0_cs35l56_link2_link3[] =
 	{}
 };
 
+/*
+ * Pretend machine quirk. The argument type is not the traditional
+ * 'struct snd_soc_acpi_mach' pointer but instead the sdw_intel_ctx
+ * which contains the peripheral information required for the
+ * SoundWire/SDCA filter on the SMART_MIC setup and interface
+ * revision. When the return value is false, the entry in the
+ * 'snd_soc_acpi_mach' table needs to be skipped.
+ */
+static bool is_device_rt712_vb(void *arg)
```

Comment:
I am thinking will it be better to add `is_device_rt712_vb` in a generic file? LNL+ platforms will need this, too.

Reply:
good suggestion @bardliao, will move the code.